ci-operator/step-registry/openshift/e2e/openstack/ipi/openshift-e2e-openstack-ipi-workflow.yaml

```diff
@@ -0,0 +1,18 @@
+workflow:
+  as: openshift-e2e-openstack-ipi
+  steps:
+    pre:
+    - chain: ipi-openstack-pre
+    post:
+    - chain: ipi-openstack-post
+    env:
+      OPENSTACK_EXTERNAL_NETWORK: "external"
+      OS_CLOUD: "openstack-cloud"
+      OPENSTACK_COMPUTE_FLAVOR: "m1.s2.xlarge"
+      CONFIG_API_VERSION: "v1beta4"
```

Comment:
v1beta4 seems like an oversight. I know it's like that in the current templates, but would it be possible to leave this out and use the default value set in https://github.com/openshift/release/pull/11000/files#diff-a5cf4017a452b7856e949eeaa4afe2f9R18 ?

Reply:
yes! that is probably best.